archivematica/startTransfer.py

```diff
@@ -28,8 +25,16 @@
     # start transfer: https://wiki.archivematica.org/Archivematica_API#Start_Transfer
     start = requests.post(full_url, headers=headers, data=params)
     print(start.json()['message'] + time.strftime(" %b %d %H:%M:%S"))
-    time.sleep(30) # pause for 30 seconds
+    time.sleep(5)
     print("Approving {}".format(txfr) + time.strftime(" %b %d %H:%M:%S"))
+    while True:
+        unapproved_transfers = requests.get(os.path.join(baseurl, 'transfer/unapproved'), headers=headers).json().get('results')
+        if unapproved_transfers:
+            print("There are unapproved transfers!")
+            break
+        else:
+            print("No transfers awaiting approval")
+            time.sleep(5)
```

Comment:
Idly wondering if it makes sense to extract this sleep time to a config setting?

Reply:
Do you mean as an external config, or a variable that's set once?

Reply:
Probably a variable that's set once, since you don't have an external config set up already.